tests/credential_plugins_test.py

```diff
@@ -137,14 +137,13 @@ class TestDelineaImports:
     library, so these tests are designed to fail if these wind up using the
     fallback import."""
 
-    def test_dsv_import(self):
+    def test_dsv_import(self) -> None:
         from awx_plugins.credentials.dsv import SecretsVault  # noqa
         # assert this module as opposed to older thycotic.secrets.vault
         assert SecretsVault.__module__ == 'delinea.secrets.vault'
 
-    def test_tss_import(self):
+    def test_tss_import(self) -> None:
         from awx_plugins.credentials.tss import DomainPasswordGrantAuthorizer, PasswordGrantAuthorizer, SecretServer, ServerSecret  # noqa
```

Comment:
Keep that original separator line
```suggestion
        from awx_plugins.credentials.tss import DomainPasswordGrantAuthorizer, PasswordGrantAuthorizer, SecretServer, ServerSecret  # noqa

```